omega/omega-spring-starter/src/main/java/org/apache/servicecomb/pack/omega/spring/OmegaSpringConfig.java

```diff
@@ -77,20 +86,28 @@ ParametersContext parametersContext() {
     return new DefaultParametersContext();
   }
 
+  @Bean
+  @ConditionalOnProperty(name = "alpha.cluster.register.type", havingValue = "default", matchIfMissing = true)
+  AlphaClusterDiscovery alphaClusterAddress(@Value("${alpha.cluster.address:localhost:8080}") String[] addresses){
+    return AlphaClusterDiscovery.builder().addresses(addresses).build();
+  }
+
   @Bean
   AlphaClusterConfig alphaClusterConfig(
-      @Value("${alpha.cluster.address:localhost:8080}") String[] addresses,
       @Value("${alpha.cluster.ssl.enable:false}") boolean enableSSL,
       @Value("${alpha.cluster.ssl.mutualAuth:false}") boolean mutualAuth,
       @Value("${alpha.cluster.ssl.cert:client.crt}") String cert,
       @Value("${alpha.cluster.ssl.key:client.pem}") String key,
       @Value("${alpha.cluster.ssl.certChain:ca.crt}") String certChain,
+      @Lazy AlphaClusterDiscovery alphaClusterDiscovery,
       @Lazy MessageHandler handler,
       @Lazy TccMessageHandler tccMessageHandler) {
 
+    LOG.info("Discovery alpha cluster address from {} {}",alphaClusterDiscovery.getDiscoveryType().name()
+            ,alphaClusterDiscovery.getDiscoveryInfo() == null ? "" : alphaClusterDiscovery.getDiscoveryInfo());
     MessageFormat messageFormat = new KryoMessageFormat();
     AlphaClusterConfig clusterConfig = AlphaClusterConfig.builder()
-        .addresses(ImmutableList.copyOf(addresses))
+        .addresses(ImmutableList.copyOf(alphaClusterDiscovery.getAddresses()))
```

Comment:
How can we implement the dynamic address lookup? I think we need to add the alphaClusterDiscovery address here.